### PR TITLE
Create script for running functional tests.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Add script to functional test to run tests.
PR to almighty was created: https://github.com/openshiftio/openshiftio-cico-jobs/pull/882

### How have you tested this PR?
Locally against prod and prod-preview.

### CODE REVIEW

File: functional-tests/devscripts/prepare_env.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+yum install docker
```
Comment (amisevsk) [inline functional-tests/devscripts/prepare_env.sh]: Does this need to be `yum install -y docker`?

File: functional-tests/devscripts/prepare_env.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+systemctl start docker
+docker pull quay.io/openshiftio/rhchestage-rh-che-functional-tests-dep
```
Comment (ibuziuk) [inline functional-tests/devscripts/prepare_env.sh]: Please, add new line in the end

File: functional-tests/devscripts/run_tests.sh [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	docker run --name functional-tests-dep --privileged \
+	           -v /var/run/docker.sock:/var/run/docker.sock \
+	           -e "RHCHE_ACC_USERNAME=$USERNAME" \
+	           -e "RHCHE_ACC_PASSWORD=$PASSWORD" \
+	           -e "RHCHE_ACC_EMAIL=$EMAIL" \
+	           -e "RHCHE_ACC_TOKEN=$OFFLINE_TOKEN" \
+	           -e "RHCHE_OFFLINE_ACCESS_EXCHANGE=https://auth.prod-preview.openshift.io/api/token/refresh" \
+	           -e "RHCHE_GITHUB_EXCHANGE=https://auth.prod-preview.openshift.io/api/token?for=https://github.com" \
+	           -e "RHCHE_OPENSHIFT_TOKEN_URL=https://sso.prod-preview.openshift.io/auth/realms/fabric8/broker" \
+	           -e "RHCHE_HOST_URL=che.prod-preview.openshift.io" \
```
Comment (rhopp) [inline functional-tests/devscripts/run_tests.sh]: Shouldn't here be $HOST_URL? This way we would be able to run tests using this script also against other deployments of rhche

File: functional-tests/devscripts/prepare_env.sh (hunk 1 above)
Comment (Katka92) [reply functional-tests/devscripts/prepare_env.sh]: Yes, sure, fixed. Thank you.

**Pull request merged**